pandas/_libs/src/vendored/ujson/python/objToJSON.c

```diff
@@ -373,6 +373,24 @@ static char *PyTimeToJSON(JSOBJ _obj, JSONTypeContext *tc, size_t *outLen) {
   return outValue;
 }
 
+static char *PyDecimalToUTF8Callback(JSOBJ _obj, JSONTypeContext *tc,
+                                     size_t *len) {
+  PyObject *obj = (PyObject *)_obj;
+  PyObject *format_spec = PyUnicode_FromStringAndSize("f", 1);
+  PyObject *str = PyObject_Format(obj, format_spec);
+  Py_DECREF(format_spec);
+
+  if (str == NULL) {
+    *len = 0;
```

Comment:
Minor but I think can also leave this untouched - no reason to say anything about the length here in case of a failure